athletics.lic

```diff
@@ -373,10 +376,7 @@ class Athletics
       walk_to(13_117)
       walk_to(6443)
       break if done_training?
-      walk_to(@outdoorsmanship_rooms.sample) unless @outdoorsmanship_rooms.empty?
-      stow_athletics_items
-      wait_for_script_to_complete('outdoorsmanship', ['3'])
-      get_athletics_items
+      outdoorsmanship_waiting(3)
```

Comment:
Noticed the other sections that call this function were passing in 4 but this one passes in 3. Is that on purpose? If not, suggesting we change it to 4 for consistency (or, remove the numerical argument and have the method just do 4 rounds or some configurable value that's passed in to the script or read from yaml). Thoughts?
```suggestion
      outdoorsmanship_waiting(4)
```

Reply:
It's a very long climb run so more than 15 seconds have passed between the time the first climb occurs and the time you get back to the spot where it runs the outdoorsmanship section so it only collects 3 times.